docs/source/configuration_checklist.rst

```diff
@@ -0,0 +1,67 @@
+Configuring a new project
+===============================================================================
+
+So you've created a new python project from the template! Great!
+
+Now let's get other services linked up to your new project to make the most of
+what the template and open source CI has to offer!
+
+Configure your GitHub repository for safety and security
+-------------------------------------------------------------------------------
+
+* Consider setting up branch protection rules.
+
+  * `GitHub Instructions for protected branches <https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#require-pull-request-reviews-before-merging>`_
+  * This will help ensure your code is always ready to deploy by running all tests
+    pass to merging into the ``main`` branch.
```

Comment:
This sentence, sense does not make. 

```suggestion
    before merging into the ``main`` branch.
```

Reply:
I was probably the person who wrote it :(

Reply:
I mean... not to blame anyone. But yes. Blame says it was you =D

https://github.com/lincc-frameworks/python-project-template/commit/76580c43e77f319b195c770cc942579e64a4f0a9

Reply:
Ugh, you just can't hide from git.